core/ui/src/main/kotlin/com/google/samples/apps/nowinandroid/core/ui/NewsResourceCard.kt

```diff
@@ -112,7 +128,22 @@ fun NewsResourceCardExpanded(
                     Row {
                         NewsResourceTitle(
                             userNewsResource.title,
-                            modifier = Modifier.fillMaxWidth((.8f)),
+                            modifier = Modifier.fillMaxWidth((.8f))
+                                .dragAndDropSource {
+                                    detectTapGestures(
+                                        onLongPress = {
```

Comment:
Is there a test we can add to verify this?

Reply:
Unfortunately, not. We have no testing APIs for this

Reply:
Is there a bug to track for those?